infobat/util.py

```diff
@@ -45,28 +45,43 @@ def delta_to_string(_, delta):
     return timestr
 
 _time_coefficients = {
-    's': 'seconds',
-    'm': 'minutes',
-    'h': 'hours',
-    'd': 'days',
-    'w': 'weeks', 'wk': 'weeks',
-    'mo': 'months',
-    'y': 'years', 'yr': 'years',
+    's': 'seconds', 'second': 'seconds', 'seconds': 'seconds',
+    'min': 'minutes', 'minute': 'minutes', 'minutes': 'minutes',
+    'h': 'hours', 'hour': 'hours', 'hours': 'hours',
+    'd': 'days', 'day': 'days', 'days': 'days',
+    'w': 'weeks', 'wk': 'weeks', 'week': 'weeks', 'weeks': 'weeks',
+    'mo': 'months', 'month': 'months', 'months': 'months',
+    'y': 'years', 'yr': 'years', 'year': 'years', 'years': 'years',
 }
```

Comment:
Hard to read since this abuts the function def. Rename to uppercase, and add some whitespace.